### DISCUSSION TITLE
Solo Farm

### DISCUSSION THREAD

**Discussion by Ratchanida** [Chia-Network/chia-blockchain#7318] [Support]
I've updated a new pool protocol (chia blockchain version 1.2.0)   I chose option 1: Solo Pool.  I have 12 plots NFT done but my Solo pool screen shows 0 plot.  Have I done something wrong?

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
